### PR TITLE
[RFC] treewide: remove useless ucitrack definitions

### PR DESCRIPTION
**Pull request by ysc3839**
#3381

### CODE REVIEW

File: applications/luci-app-adblock/root/etc/uci-defaults/40_luci-adblock [hunk 1]
```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
 
-uci -q batch <<-EOF >/dev/null
```
Comment (ysc3839) [inline applications/luci-app-adblock/root/etc/uci-defaults/40_luci-adblock]: Should I delete that ucitrack config like
https://github.com/openwrt/luci/blob/74cb64fbb7bf0c6a3c26ac7b0388ff0b01649d5a/applications/luci-app-ddns/root/etc/uci-defaults/40_luci-ddns#L3-L7

File: applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon [hunk 2]
```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
 
-uci -q batch <<-EOF >/dev/null
```
Comment (ysc3839) [inline applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon]: https://github.com/openwrt/packages/blob/5da8ae16b23293250ac6c28bd727101b9c539500/net/nlbwmon/files/nlbwmon.init#L102
nlbwmon doesn't contain a trigger for config `nlbwmon`, only for `dhcp` and `system`. Should I keep this?

File: applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass [hunk 3]
```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
-uci -q batch <<-EOF >/dev/null
```
Comment (ysc3839) [inline applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass]: https://github.com/openwrt/packages/blob/5da8ae16b23293250ac6c28bd727101b9c539500/net/vpnbypass/files/vpnbypass.init#L115
vpnbypass doesn't contain a trigger for config `vpnbypass`. Should I keep this?

**Comment by feckert** [comment]
I think we could remove this file complete if the generic part 

> rm -f /tmp/luci-indexcache

could be moved into https://github.com/openwrt/luci/blob/master/luci.mk#L218

In files where other uci commands are issued, only the ucitrack part should of course be removed.

File: applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass (hunk 3 above)
Comment (feckert) [reply applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass]: good question.
Maybe we should add the vpnbypass file to the trigger service?

**Comment by feckert** [comment]
And we should split the into separate commits.
So we are able to cherry-pick them.

File: applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon (hunk 2 above)
Comment (feckert) [reply applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon]: good question.
Maybe we should add the vpnbypass file to the trigger service?

Comment (ysc3839) [reply applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon]: @jow-

**Comment by ysc3839** [comment]
Any updates? What should I do with `uci-defaults`? @jow-

File: applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon (hunk 2 above)
Comment (jow-) [reply applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon]: Please add a trigger for the nlbwmon config

File: applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass (hunk 3 above)
Comment (jow-) [reply applications/luci-app-vpnbypass/root/etc/uci-defaults/40_luci-vpnbypass]: Please add a trigger for the `vpnbypass` uci config

**Comment by ysc3839** [comment]
@jow- nlbwmon and vpnbypass are fixed, what's next?

**Comment by jow-** [comment]
Merged via 814ad96df9634f39cb793c31d55b0351f88fadff - thanks!

**Pull request closed without merging**